app/uk/gov/hmrc/fileupload/quarantine/events.scala

```diff
@@ -20,7 +20,7 @@ import play.api.libs.json.{Format, JsObject, Json}
 import uk.gov.hmrc.fileupload.{EnvelopeId, Event, FileId, FileRefId}
 
 case class FileInQuarantineStored(envelopeId: EnvelopeId, fileId: FileId, fileRefId: FileRefId,
-                                  created: Long, name: String, contentType: String, metadata: JsObject) extends Event
+                                  created: Long, name: String, fileLength: Long, contentType: String, metadata: JsObject) extends Event
```

Comment:
this fileLength will be used at the backend, the backend does the finial checks when the file is in quarantine, those checks need to know the fileLength of a file and that is come from here (if am right).